.gitignore

```diff
@@ -3,4 +3,5 @@ package-lock.json
 __pycache__
 .idea
 .DS_Store
-.env
\ No newline at end of file
+.env
+celerybeat-schedule.db
```

Comment:
I noticed this file added in ".gitignore", but you didn't add in the commit.
What is the file for? Is this file automatically generated? or we need to add this to our repository as well?

Reply:
this is an automatically generated db file by celery which we dont need to track. see my previous [commit](https://github.com/CSC510-GROUP-40/FitnessApp/pull/4/commits/2bf55c51805437be7a58cab7dd8f53ef52cf877e)